### ISSUE TITLE
Accession for closest match

### ISSUE THREAD

**Issue by cliffbueno** [SushiLab/mTAGs#4]
Thanks for a great tool! I was wondering if an any of the output files it is stated anywhere the accession for the closest match in the SILVA database to each of the extracted SSU sequences from my metagenomes?

**Comment by hjruscheweyh** [comment]
Hi @cliffbueno 

Thank you for using mTAGs.

mTAGs aligns rRNA sequences against the SILVA database using vsearch. This produces, as an intermediate step, the m8 alignment files which hold the alignment information of each rRNA sequence against the SILVA database. You should be able to filter the m8 files for that purpose. 

Feel free to check and report back.

Hans

**Comment by cliffbueno** [comment]
Thank you. I think I figured it out! I see the m8 files. There are also multiple .m8 files per sample, some with .pe and some with .se prefix. As far as I can tell, the multiple files were to keep the input .fastas to 2000 lines max. In the m8 files I can see that there is a list containing the matches to your SILVA OTU IDs. I read that "The otu level entries represent taxonomic units that were generated by complete linkage clustering of distances between all sequences inside the respective genus." But this OTU ID is different than an accession. However, since I was really just interested in obtaining the reference sequence, I figured I could extract that from the downloaded mtags database based on the OTU ID. I found the path to the database and saw it was a .udb file but was able to use vsearch --udb2fasta to convert it to fasta. Now I see the sequences for each of the OTU IDs. Thanks!
